ansible/roles/slurm/tasks/main.yml

```diff
@@ -67,6 +67,7 @@
   - conf.py
   - resume.py
   - setup.py
+  - setup_network_storage.py
```

Comment:
Just confirming that this will work without an updated image. Because the new file is in the scripts dir that devel is already placing on machines. If your understanding is the same then close this.

Reply:
Yes, I am not building new image now. Put more details in the description.